### ISSUE TITLE
Travis build is failing

### ISSUE THREAD

**Issue by siddharthist** [GaloisInc/saw-script#436]
Here's the log:
```
Configuring boomerang-1.3.1...
Preprocessing library for boomerang-1.3.1..
Building library for boomerang-1.3.1..
[1 of 9] Compiling Text.Boomerang.HStack ( Text/Boomerang/HStack.hs, dist/build/Text/Boomerang/HStack.o )
[2 of 9] Compiling Text.Boomerang.Pos ( Text/Boomerang/Pos.hs, dist/build/Text/Boomerang/Pos.o )
[3 of 9] Compiling Text.Boomerang.Prim ( Text/Boomerang/Prim.hs, dist/build/Text/Boomerang/Prim.o )
Text/Boomerang/Prim.hs:57:10: error:
    • No instance for (Applicative (Parser e tok))
        arising from the superclasses of an instance declaration
    • In the instance declaration for ‘Monad (Parser e tok)’
   |
57 | instance Monad (Parser e tok) where
   |          ^^^^^^^^^^^^^^^^^^^^
```
@robdockins You fixed a similar error with Crucible recently, right? Will this get fixed if we update the submodule?

That version of boomerang is quite old: https://hackage.haskell.org/package/boomerang

**Comment by robdockins** [comment]
Yeah, this is a weird one.  The fix was to increase package lower bounds in various dependencies.  Probably if you pull in the crucible submodule it will fix this.

**Comment by siddharthist** [comment]
#425 now updates the Crucible submodule, let's see if that does the trick.

**Comment by siddharthist** [comment]
Fixed in #425
https://travis-ci.org/GaloisInc/saw-script/builds/532613545
